server/.config_variable

```diff
@@ -0,0 +1,56 @@
+# Serveur
+export NODE_ENV="production"
+export PORT=8000
+export HOST="critick.v1"
+export PROTOCOL="http" 
+
+# Base de données
+export DATABASE_TYPE="postgresql"
```

Comment:
like i say, we want the database confidential so think about one way to setup them manually

Reply:
ok ok